static/app/utils/profiling/hooks/useProfileFilters.tsx

```diff
@@ -0,0 +1,31 @@
+import {useEffect, useState} from 'react';
+
+import {Client} from 'sentry/api';
+import {Organization, Tag} from 'sentry/types';
+import useApi from 'sentry/utils/useApi';
+import useOrganization from 'sentry/utils/useOrganization';
+
+type ProfileFilters = {[key: string]: Tag};
+
+function useProfileFilters(): ProfileFilters {
```

Comment:
We may need to expose things like {isLoading, error} etc at some point, all these custom async hooks are not going to scale well across the codebase.